language/Types.md

```diff
@@ -74,7 +74,16 @@ instance showPercentage :: Show Percentage where
 
 ## Functions
 
-Functions in PureScript are like their Javascript counterparts, but always have exactly one argument.
+Functions in PureScript are like their Javascript counterparts, but always have exactly one argument. Function declarations and lambdas (anonymous functions) with multiple arguments are shorthand for curried lambda abstractions. For example, the following are all equivalent:
+
+```purescript
+add x y = x + y
+add = \x y -> x + y
+add = \x -> \y -> x + y
+add = (+)
+```
+
+Despite this overwhelming convention, uncurried JavaScript functions are can be used through the [purescript-functions](https://pursuit.purescript.org/packages/purescript-functions/) library. For more information see [The Foreign Function Interface](http://www.purescript.org/learn/ffi/).
```

Comment:
I think this new paragraph is a bit outside the scope of this file; I'm not sure this should be added here.